openedx/features/edly/utils.py

```diff
@@ -1,5 +1,34 @@
 import jwt
 from django.conf import settings
+from django.contrib.sites.shortcuts import get_current_site
+from django.db.models import Q
+
+from openedx.features.edly.models import EdlySubOrganization
+
+
+def user_has_edly_organization_access(request):
+    """
+    Returns True if the edly sub organization of the requested URL belongs to the user's linked edly sub organizations.
```

Comment:
Ideally the docstring shouldn't exceed 80 characters length 
https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#docstrings
```python
Check if the requested URL site is allowed for the user.

This method checks if the requested URL site is in the requesting user's
edly sub organizations list.
```

Reply:
Done